docs/3.api/2.composables/use-route.md

```diff
@@ -44,4 +44,8 @@ Apart from dynamic parameters and query parameters, `useRoute()` also provides t
 - `path`: encoded pathname section of the URL
 - `redirectedFrom`: route location that was attempted to access before ending up on the current route location
 
+::callout
+The router isn't able to detect if there is an anchor in the path server-side. Using the `fullPath` in the template can trigger hydration issues.
```

Comment:
```suggestion
Browsers don't send URL fragments (for example `#foo`) when making requests. So using `route.fullPath` in your template can trigger hydration issues because this will include the fragment on client but not the server.
```